Update readme with search overlay (#30)
Relewise/relewise-ui-components: packages/web-components/README.md

                .addProductAssortmentFilter(1);
        },
    }
});
```

#### Product Search Overlay
This component renders a search bar that will [search for products](https://docs.relewise.com/docs/intro/search.html#product-search) in Relewise and show results in an overlay.

```html
<relewise-product-search-overlay displayed-at-location="LOCATION"></relewise-product-search-overlay>
```
The component renders products but does not use the default product template.

To overwrite the template used in this specific component call the useSearch function with the templates option.
```ts
useSearch({
    filters: { ... },
    templates: {
        searchOverlayProductResult: (product, { html, helpers }) => {
            return html`<!-- Write your template here -->`;
        },
    }
});
```

##### Attributes

- **displayed-at-location** : 
    
    Where the search bar is being shown. 
    
    For more information see our [docs](https://docs.relewise.com/docs/developer/bestpractice.html#_6-making-search-requests).

- **search-bar-placeholder** (Optional, *Default "Search"*):
    
    The placeholder used in the search bar.

- **number-of-products** (Optional, *Default 5*): 

>     The number of products shown i the result overlay.

Reviewer: ```suggestion
    The number of products shown in the results overlay.
```

- **number-of-search-term-predictions** (Optional, *Default 3*): 

>     The number of search term predictions shown i the result overlay.

Reviewer: ```suggestion
    The number of search term predictions shown in the results overlay.
```

- **no-results-message** (Optional, *Default "No search results found"*): 

>     The message shown in the overlay when no results found.

Reviewer: ```suggestion
    The message shown in the overlay when there are no results found.
```

- **debounce-time** (Optional, *Default 250*): 

>     The amount of delay between inputs in milliseconds before requesting Relewise with a new search call.

Reviewer: ```suggestion
    The amount of time, in milliseconds, that must pass between requests to Relewise with a new search call.
```
